must-call-checker/src/main/java/org/checkerframework/checker/mustcall/MustCallAnnotatedTypeFactory.java

```diff
@@ -64,9 +65,20 @@ public MustCallAnnotatedTypeFactory(final BaseTypeChecker checker) {
     BOTTOM = createMustCall();
     POLY = AnnotationBuilder.fromClass(elements, PolyMustCall.class);
     addAliasedAnnotation(InheritableMustCall.class, MustCall.class, true);
+    addAliasedAnnotation(MustCallChoice.class, POLY);
```

Comment:
What was the final strategy we went with for the `@MustCallChoice` annotation?  As an alias, or as a separate declaration annotation?

Reply:
It is an alias. In the end that was the only strategy that I could get working reliably.